test/src/main/java/org/apache/accumulo/test/ManagerRepairsDualAssignmentIT.java

```diff
@@ -99,12 +102,21 @@ public void test() throws Exception {
         }
       }
       assertEquals(2, states.size());
+      int deadCount = getZkDeadCount(cluster.getServerContext().getZooReader(),
+          cluster.getServerContext().getInstanceID());
```

Comment:
This should always be 0 at the start of the test.

Reply:
It should - but I wanted to avoid the assumption in case there was any chance of a concurrent test had / left something there - a very remote possibility. It just seemed more general to get the count. kill a server and wait for that count to increase.

Reply:
If a concurrent test was somehow using this test's instance of MiniAccumuloCluster, that'd be a big problem, likely to be the cause of other test flakes. So, we'd want to detect that, not obscure it by progressing with a different count that was detected. I would just add a simple `assertEquals(0, actual)` here instead of trying to record and track the previous number.